build/calico-cni-plugin/Dockerfile

```diff
@@ -0,0 +1,10 @@
+# Equal to centos:7.6.1810
+# SHA256 digest of the base image
+# TODO: centrally define a base image
+ARG BUILD_IMAGE_SHA256=5d4f4e6051c7cc10f2e712f9dc3f86a2bd67e457bced7ca52a71c243099c0121
+ARG BUILD_IMAGE=docker.io/centos
+FROM ${BUILD_IMAGE}@sha256:${BUILD_IMAGE_SHA256} as build
+
+RUN yum install -y yum-utils rpm-build rpmlint rpmdevtools
```

Comment:
Two remarks:

- Consider using something like
```
RUN yum install -y \
    rpm-build \
    rpmdevtools \
    rpmlint \
    yum-utils
```
which reduces the size of a diff when a tool needs to be added (adding a single line instead of replacing one is easier to review)

- Add `&& yum clean all` at the end of the `RUN` stanza.